docs/multi_process.md

```diff
@@ -0,0 +1,127 @@
+*   Each process has a distinct set of _local_ devices it can address. The
+    _global_ devices are the set of all devices across all processes.
+*   Use standard JAX parallelism APIs like {func}`~jax.pmap` and
+    {func}`~jax.experimental.maps.xmap`. Each process “sees” _local_ input and
+    output to parallelized functions, but communication inside the computations
+    is _global_.
+*   Make sure all processes run the same parallel computations in the same
+    order.
+
+
+### Launching JAX processes
+
+Unlike other distributed systems where a single controller node manages many
+worker nodes, JAX uses a “multi-controller” programming model where each JAX
+Python process runs independently, sometimes referred to as an [Single Program,
+Multiple Data (SPMD)](https://en.wikipedia.org/wiki/SPMD) model. Generally, the
+same JAX Python program is run in each process, with only slight differences
+between each process’s execution (e.g. different processes will load different
+input data). Furthermore, **you must manually run your JAX program on each
+host!** I.e., JAX doesn’t automatically start multiple processes from a single
+program invocation.
+
+(This is why this guide isn’t offered as a notebook -- we don’t currently have a
+good way to manage multiple Python processes from a single notebook.)
+
+
+### Local vs. global devices
+
+Before we get to running multi-process computations from your program, it’s
+important to understand the distinction between _local_ and _global_ devices.
+
+**A process’s _local_ devices are those that it can directly address and launch
+computations on.** For example, in a Cloud TPU pod, each host can only launch
+computations on the 8 TPU cores attached directly to that host (see the [Cloud
+TPU System Architecture](https://cloud.google.com/tpu/docs/system-architecture)
+documentation for more details). You can see a process’s local devices via
+{func}`jax.local_devices()`.
+
+**The _global_ devices are the devices across all processes.** A computation can
+span devices across processes and perform collective operations via the direct
+communication links between devices, as long as each process launches the
+computation on its local devices. You can see all available global devices via
+{func}`jax.devices()`. Note that a process’s local devices are always a subset
+of the global devices.
+
+
+### Running multi-process computations
+
+So how do you actually run a computation involving cross-process communication?
+**Use the same parallel evaluation APIs that you would in a single process!**
+
+For example, {func}`~jax.pmap` can be used to run a parallel computation across
+multiple processes. (If you’re not already familiar with how to use
+{func}`~jax.pmap` to run across multiple devices within a single process, check
+out the {doc}`/jax-101/06-parallelism` notebook.) Each process should call the
+same pmapped function and pass in arguments to be mapped across its _local_
```

Comment:
Question (newbie) @skye : is "pmapped" the same "jitted" - part of the common JAX lingo? If so, then ignore the following suggestion: "`pmap`ped function" 😄 

```suggestion
same `pmap`ped function and pass in arguments to be mapped across its _local_
```

Reply:
Hm `pmap`ped is probably more accurate, but I'm gonna claim pmapped as official lingo because it looks much nicer :)

Reply:
@skye It's also cooler! pmapped 🚀 xmapped 🚀 (@saeta)